sw/device/tests/spi_host_smoketest.c

```diff
@@ -155,7 +155,7 @@ status_t test_quad_read(void) {
 static bool is_4_bytes_address_mode_supported(void) {
   enum { kSupportOnly3Bytes, kSupport3and4Bytes, kSupportOnly4Bytes };
   uint8_t address_mode =
-      bitfield_field32_read(sfdp.bfpt[0], SPI_FLASH_ADDRESS_MODE);
+      (uint8_t)bitfield_field32_read(sfdp.bfpt[0], SPI_FLASH_ADDRESS_MODE);
```

Comment:
I would just change the type of `address_mode`.

Reply:
Done!